src/main/java/com/github/steveice10/mc/protocol/data/message/ItemHoverMessage.java

```diff
@@ -0,0 +1,72 @@
+package com.github.steveice10.mc.protocol.data.message;
+
+import com.github.steveice10.mc.protocol.data.game.entity.metadata.ItemStack;
+import com.github.steveice10.mc.protocol.data.message.style.MessageStyle;
+import com.github.steveice10.opennbt.tag.builtin.CompoundTag;
+import com.google.gson.JsonElement;
+import lombok.EqualsAndHashCode;
+import lombok.NonNull;
+
+import java.util.List;
+
+@EqualsAndHashCode(callSuper = true)
+public class ItemHoverMessage extends Message {
+    public static class Builder extends Message.Builder<Builder, ItemHoverMessage> {
+        @NonNull
+        private String id = "";
+        private int count;
+        private JsonElement tag;
+
+        public Builder id(@NonNull String id) {
+            this.id = id;
+            return this;
+        }
+
+        public Builder count(@NonNull int count) {
```

Comment:
int can't be null, so NonNull is not needed here.

Reply:
Fixed